src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -420,6 +422,18 @@ const std::map<cmp_enum, int> cmpToIndex = {
                     throw std::invalid_argument("Actionx: " + actx.name());
                 }
 
+                /* item[15] is a parameter that indicates whether left_paren or right_paren is used in an expression
+                 * = 0  : no open_paren or left_paren, or both open_paren and right_paren
+                 * = 1  : left_paren at start of condition
+                 * = 2  : right_paren at end of condition
+                */
+
+                if ((cond_it->left_paren) && (!cond_it->right_paren)) {
+                    iAcn[ind + 15] = 1;
```

Comment:
The integer literals 15 and 17 should get proper names.

Reply:
Possible alternative:
```C++
if (cond_it->left_paren != cond_it->right_paren) {
     if (cond_it->left_paren)
           iAcn[ind + 15] = 1;
     else
          iAcn[ind + 15] = 2;
}
```

Reply:
I have given the two integers literals 15 and 17 names, and used the functions open_paren() and close_paren() to simplify the code.